kola/tests/docker/docker.go

```diff
@@ -64,6 +65,18 @@ func init() {
 
 		MinVersion: semver.Version{Major: 1192},
 	})
+	register.Register(&register.Test{
```

Comment:
Right now we don't have any way to limit architectures in Kola (that I know of, I asked around a bit before committing this)